.github/workflows/e2e.yml

```diff
@@ -0,0 +1,80 @@
+name: Test e2e
+
+on:
+  pull_request:
+    types:
+      - opened
+    branches:
+      - develop
+
+jobs:
+  test-e2e:
+    runs-on: ubuntu-latest
+    env:
+      FIFTYONE_DO_NOT_TRACK: true
+      ELECTRON_EXTRA_LAUNCH_ARGS: "--disable-gpu"
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip setuptools wheel
+
+      - name: Cache Node Modules
+        id: node-cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            app/node_modules
+            app/.yarn/cache
+          key: node-modules-${{ hashFiles('app/yarn.lock') }}
+
+      - name: Install app
+        if: steps.node-cache.outputs.cache-hit != 'true'
+        run: cd app && yarn install
+
+      - name: Build app
+        run: make app
+
+      - name: Install fiftyone
+        run: pip install .
+
+      - name: Configure
+        id: test_config
+        run: |
+          python tests/utils/setup_config.py
+          python tests/utils/github_actions_flags.py
+
+      - name: FFmpeg
+        uses: FedericoCarboni/setup-ffmpeg@v2
+
+      - name: Run e2e tests
+        run: |
+          FIFTYONE_DATABASE_NAME=cypress python ../fiftyone/server/main.py --address 0.0.0.0 --port 8787 &
```

Comment:
@benjaminpkane Is this supposed to replace the fake server? 🥳  that's cool, but `yarn start` will no longer start cypress correctly locally, no?

Reply:
I guess we could add something like:
```
"cy:run": "FIFTYONE_DATABASE_NAME=cypress python ../fiftyone/server/main.py --address 0.0.0.0 --port 8787 & cypress run"
```

Reply:
Yes, I'll tweak it

Reply:
I'm proposing the background server is started manually by the developer when testing locally with `yarn start-test-server`. I updated the readme. Trying to avoid creating orphaned processes